blockchain-importer/src/Pos/BlockchainImporter/Tables/TxsTable.hs

```diff
@@ -63,37 +61,46 @@ txsTable = Table "txs" (pTxs TxRow  { trHash            = required "hash"
                                     , trOutputsAmount   = required "outputs_amount"
                                     , trBlockNum        = required "block_num"
                                     , trTime            = required "time"
+                                    , trSucceeded       = required "succeeded"
                                     })
 
 txAddrTable :: Table TxAddrRowPGW TxAddrRowPGR
 txAddrTable = transactionAddrTable "tx_addresses"
 
+insertConfirmedTx :: Tx -> TxExtra -> Word64 -> PGS.Connection -> IO ()
```

Comment:
lift a comment related to the decision of stored a `succeed` column (it could be a virtual field if we normalize the DB)